plugins/dashboard/__init__.py

```diff
@@ -651,6 +652,10 @@ def label(self):
     def to_configure_plot_params(self):
         return {**self.raw_params, "name": self.name}
 
+    def to_cache_key(self):
+        key = json.dumps(self.raw_params, sort_keys=True)
```

Comment:
Is there some well known JSON => key algorithm out there? this seems a bit too simplistic.